luaui/barwidgets.lua

```diff
@@ -594,11 +545,6 @@ function widgetHandler:LoadWidget(filename, fromZip, enableLocalsAccess)
 		local md5 = VFS.CalculateHash(text,0)
 		if widgetHandler.widgetHashes[md5] == nil then
 			widgetHandler.widgetHashes[md5] = filename
-			-- Embed LuaRules message that we enabled a new user widget
-			--local success, err = pcall(widgetHandler.AddSpadsMessage, widgetHandler, tostring(filename) .. ":" .. tostring(md5))
-			--if success == false then
-			--	Spring.Echo("widgetHandler.AddSpadsMessage call failed", tostring(err))
-			--end
```

Comment:
This was "temporarily" disabled 6 months ago. If we would like to keep track of this code, we can make a PR to add it back in.